std_msgs/Header header
int32 uav_id_from
int32 issue
State state
float64 duration
PathData[] path_coord
MyBspline bspline
bezierCurve bzCurve